process/smartContract/process.go

```diff
@@ -541,19 +571,27 @@ func (sc *scProcessor) refundGasToSender(
 }
 
 // save account changes in state from vmOutput - protected by VM - every output can be treated as is.
-func (sc *scProcessor) processSCOutputAccounts(outputAccounts []*vmcommon.OutputAccount) ([]*vmcommon.OutputAccount, error) {
-	crossOutAccs := make([]*vmcommon.OutputAccount, 0)
+func (sc *scProcessor) processSCOutputAccounts(outputAccounts []*vmcommon.OutputAccount, tx *transaction.Transaction) error {
+	sumOfAllDiff := big.NewInt(0)
+	sumOfAllDiff = sumOfAllDiff.Sub(sumOfAllDiff, tx.Value)
+
+	zero := big.NewInt(0)
 	for i := 0; i < len(outputAccounts); i++ {
 		outAcc := outputAccounts[i]
 		acc, err := sc.getAccountFromAddress(outAcc.Address)
 		if err != nil {
-			return nil, err
+			return err
 		}
 
-		fakeAcc := sc.tempAccounts.TempAccount(outAcc.Address)
-
 		if acc == nil || acc.IsInterfaceNil() {
-			crossOutAccs = append(crossOutAccs, outAcc)
+			if outAcc.BalanceDelta != nil {
+				// TODO: remove this when VM resolved
```

Comment:
"VM resolved" ?

Reply:
IELE VM has a bug and I am waiting for the new release.